### ISSUE TITLE
100% wrong

### ISSUE THREAD

**Issue by NeverAgain329** [svanheulen/boxdestroyer-windower-addon#11]
This plugin no longer works? 

It suggests answers that completely contradict the clues given.

**Comment by svanheulen** [comment]
The `messages.lua` file probably needs to be updated. There's a script (with instructions) in the `messages` folder for re-building that file. You or someone else will need to do that since it requires the updated game and I don't play any more.

**Comment by revoemag** [comment]
[messages.lua.txt](https://github.com/svanheulen/boxdestroyer-windower-addon/files/1293103/messages.lua.txt)

I ran the python script and regenerated the messages.lua file. I had to make a small change to the script to get it to find the install location. I had to change the winreg key to:
'SOFTWARE\\\WOW6432Node\\PlayOnlineUS\\InstallFolder'.

I replaced the generated message.lua file in the addon and now receive the following error:
Lua runtime error: boxdestroyer.lua:119: attempt to index global  'offsets' (a nil value)

I have attached the messages.lua file here (renamed with .txt extension so it will attach correctly).
